certificate/views.py

```
@@ -0,0 +1,77 @@
+            certificate = MedicalCertificate.objects.get(national_code=certificate_national_code)
+
+            # بررسی اگر گواهی قابل دانلود نیست
+            if not certificate.is_downloadable:
+                return Response({"error": "این گواهی هنوز برای دانلود آماده نیست"}, status=403)
+
+            # بررسی وجود فایل PDF
+            if not os.path.exists(certificate.pdf_file_path):
+                return Response({"error": "فایل PDF یافت نشد"}, status=404)
+
+            # ارسال فایل PDF برای دانلود
+            pdf_file = open(certificate.pdf_file_path, 'rb')
```

Ruff (0.12.2)

Use a context manager for opening files

(SIM115)